new_dawn_server/locations/api/resources.py

```diff
@@ -1,5 +1,6 @@
 from django.conf.urls import url
-from new_dawn_server.locations.constants import us_city_mapping, country_list
+from new_dawn_server.locations.city_constants import us_city_mapping, country_list
```

Comment:
create a directory in locations called `/constants` and create two files for each.